packages/lab/src/pill-next/PillNext.tsx

```diff
@@ -1,24 +1,20 @@
-import { forwardRef, ComponentPropsWithoutRef, MouseEvent } from "react";
+import { forwardRef, ComponentPropsWithoutRef } from "react";
 import clsx from "clsx";
 import { useWindow } from "@salt-ds/window";
 import { useComponentCssInjection } from "@salt-ds/styles";
-import { Button, makePrefixer, useButton } from "@salt-ds/core";
+import { makePrefixer, useButton } from "@salt-ds/core";
 import pillCss from "./PillNext.css";
-import { CloseIcon } from "@salt-ds/icons";
 
 export interface PillNextProps extends ComponentPropsWithoutRef<"button"> {
   /* If true the pill will be disabled */
   disabled?: boolean;
```

Comment:
`disabled` is a button attr. I think you can replace PillNextProps with ComponentPropsWithoutRef<"button">